src/main/java/com/epam/bookstoreservice/service/impl/LoginServiceImpl.java

```diff
@@ -33,7 +33,7 @@ public class LoginServiceImpl implements LoginService {
     private static final String TOKEN_HEADER = "Bearer";
 
     @Override
-    public String loginAndReturnToken(UserRequestDTO userRequestDto) {
+    public TokenResponseDTO loginAndReturnToken(UserRequestDTO userRequestDto) {
         UserDetails userDetails = userDetailsService.loadUserByUsername(userRequestDto.getPhoneNumber());
         if (Objects.isNull(userDetails) ||
```

Comment:
Extract this validation code to a method.
Use a meaningful variable to replace the complex conditon